### PR TITLE
Add readme documentation

### LINKED ISSUE #7: create a proper readme

### PR DESCRIPTION
**Pull request by enyason**
the initial draft for readme.

closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+- Cancel predictions
+- Await prediction completion
+
+## Installation
+
+Add the following dependency to your `build.gradle` file:
+
+```groovy
+implementation 'io.github.enyason:replicate-kotlin:0.1.0'
+```
```
Comment (mayorjay) [inline README.md:36]: We can also have the kotlin version

File: README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+Reference: https://replicate.com/docs/reference/http
+
+
+## Contributing
+
+Contributions are welcome! Please read our [contributing guidelines](CONTRIBUTING.md) for more details.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
```
Comment (mayorjay) [inline README.md:161]: Do we have these files already?

File: README.md (hunk 1 above)
Comment (enyason) [reply README.md:36]: yeah, that's correct.


but the dependency name is not correct until we publish.

File: README.md (hunk 2 above)
Comment (enyason) [reply README.md:161]: No, we don't. It is just a dummy. @Lamouresparus is working on it already.

**Review by mayorjay — APPROVED** [review]

**Review by Lamouresparus — APPROVED** [review]

**Pull request merged**